modules/AOP_Case_Updates/CaseUpdatesHook.php

```diff
@@ -449,10 +469,12 @@ public function creationNotify($bean, $event, $arguments)
      * @param EmailTemplate $template
      * @param aCase $bean
      * @param $contact
+     * @param $addDelimiter
+     * @param $txtDelimiter
      *
      * @return array
      */
-    private function populateTemplate(EmailTemplate $template, aCase $bean, $contact)
+    private function populateTemplate( EmailTemplate $template, aCase $bean, $contact, $addDelimiter = true, $txtDelimiter = "" )
```

Comment:
Same as above, method signatures should not be changed unless absolutely necessary.